en/faq.md

```diff
@@ -6,110 +6,110 @@ sidebarDepth: 2
+3. Your material is cached by your skin mod;
+4. You are using the Alex model skin and your Minecraft version is lower than 1.8.
 
-解决方案分别是：
+The solutions are:
 
-1. 正确安装并配置皮肤 Mod。你可以在 [新手指引 - 配置 Mod](/newbee/mod.html) 中学习如何正确配置皮肤 Mod；
-2. 删除这些 Mod，或让它们绕过皮肤 Mod 与 LittleSkin 之间的连接；
-3. 等待几分钟后再试。如果还是没有更新，尝试清除你的皮肤 Mod 的缓存（即删除对应的文件夹）：
-    - CustomSkinLoader 的缓存位于 `.minecraft/CustomSkinLoader/caches`；
-    - SkinPort 的缓存位于 `.minecraft/cachedImages`
-4. 更换皮肤 Mod，或更换为 Steve 模型的材质：
-    - 对于 1.7.10，请使用 SkinPort 加载材质；
-    - 对于更低版本，目前无解，请更换为 Steve 模型的皮肤。
+1. Install and configure the skin mod correctly. You can learn how to properly configure the skin mod in [Beginner's Guide - Configuring Mod](/newbee/mod.html);
+2. Remove these mods, or let them bypass the connection between the skin mod and LittleSkin;
+3. Wait a few minutes and try again. If it still doesn't update, try clearing your skin mod's cache (ie deleting the corresponding folder):
+    - CustomSkinLoader's cache is located in `.minecraft/CustomSkinLoader/caches`;
+    - SkinPort's cache is located in `.minecraft/cachedImages`
+4. Change the skin mod, or change to the material of the Steve model:
+    - For 1.7.10, use SkinPort to load materials;
+    - For earlier versions, there is currently no solution, please change to the skin of the Steve model.
 
-如果你确定你的问题不是以上原因引起的，或者你按照以上的解决方案做了之后你的角色的材质依然没有显示或更新，请在详细阅读 [报告问题的正确姿势](/report.html) 后带上你的皮肤 Mod 的日志，加入 [官方用户交流群](/user-group.html) 或 [发送邮件工单](/email.html) 询问，或直接购买一对一远程技术支持服务。
+If you are sure that your problem is not caused by the above reasons, or after you follow the above solutions, the texture of your character still does not display or update, please read [Report the correct pose for problems](/report.html) Then bring your skin mod log, join the [official user exchange group](/user-group.html) or [send an email ticket](/email.html) to inquire, or directly purchase one-to-one remote technical support services .
 
-### 单人游戏中可以正常显示皮肤，但在多人游戏中就不行？
+### Skins are displayed normally in single player but not in multiplayer?
 
-这种情况往往在使用 Yggdrasil 外置登录进行服务器联机时出现。
+This situation often occurs when using Yggdrasil external login for server online.
 
-由于 Minecraft 的材质加载机制，在多人游戏中，只有服务端启用了在线模式（`online-mode`）才可以加载材质。如果你游玩的服务器没有加载 authlib-injector 并正确配置 LittleSkin 的 Yggdrasil API，那么在服务器联机时就不会加载材质。
+Due to Minecraft's texture loading mechanism, in multiplayer games, textures can only be loaded if the server has online mode (`online-mode`) enabled. If you're playing on a server that doesn't have authlib-injector loaded and LittleSkin's Yggdrasil API is configured correctly, the textures won't load when the server is online.
 
-如果你有在服务器联机中加载材质的需求，并且你是服主，请在服务端也加载并正确配置 authlib-injector；否则请使用皮肤 Mod 加载材质。
+If you need to load materials in the server online, and you are the server owner, please also load and configure the authlib-injector correctly on the server; otherwise, please use the skin mod to load the materials.
```

Comment:
```suggestion
If you need to load materials on the online server, and you are the server owner, please load and configure the authlib-injector correctly on the server; otherwise, please use the skin mod to load the materials.
```